src/core/client/stream/common/KeyboardShortcuts/KeyboardShortcuts.tsx

```diff
@@ -48,9 +45,34 @@ import { SetTraversalFocus } from "./SetTraversalFocus";
 
 import styles from "./KeyboardShortcuts.css";
 
+interface ChildComment {
+  readonly node: {
+    readonly seen: boolean | null;
+    readonly id: string;
+    readonly ancestorIDs: ReadonlyArray<string | null>;
+    readonly deleted: boolean | null;
+  };
+}
+
+interface Comment {
+  readonly node: {
+    readonly seen: boolean | null;
+    readonly id: string;
+    readonly allChildComments: {
+      readonly edges: ReadonlyArray<ChildComment>;
+    };
+    readonly deleted: boolean | null;
+    ignored?: boolean | null;
+    ignoredReplies?: string[];
+  };
+}
+
```

Comment:
Thank you for using concrete interfaces and not trying to do a `Omit<Comment>(...)` on 1000 un-needed props on the comment. This is much clearer with the exact same type safety.

I dislike the `Omit<...>` as it does the exact opposite of an interface definition, it tells me only what I don't care aboue, and does not define what I DO care about!